pkg/util/pki/csr_test.go

```diff
@@ -385,6 +386,218 @@ func TestRemoveDuplicates(t *testing.T) {
 	}
 }
 
+func makePathLen(x int) *int {
+	return &x
+}
+
```

Comment:
```suggestion
"k8s.io/utils/pointer"

pointer.Int(x)
```

Reply:
didn't know about this package, thanks :+1:

Reply:
there's no [`Int`](https://pkg.go.dev/k8s.io/utils/pointer) :sob: guess I'll change the type of MaxPathLen to int32

Reply:
Changing to use this package does mean having to cast to int32 when filling out `MaxPathLen` in `x509.Certificate`, which is of type `int`, which is a shame. I'd be interested to hear why there isn't a `pointer.Int`. Still, I'll change the CRD type to int32 anyway just for consistency across machines with different word lengths